src/createServer.js

```diff
@@ -1,3 +1,76 @@
-// Write code here
-// Also, you can create additional files in the src folder
-// and import (require) them here
+const http = require('http');
+const { convertToCase } = require('./convertToCase');
+
+const getDataFromUrl = (url, host) => {
```

Comment:
would be better to move all these function to a separate file like a `helpers.ts`